ha/core/error.py

```diff
@@ -83,4 +84,14 @@ def __init__(self, desc=None):
         _desc = '[%s] %s' %(inspect.stack()[1][3], desc)
         _message_id = HA_TEST_FAILED
         _rc = 1
-        super(HATestFailedError, self).__init__(rc=_rc, desc=_desc, message_id=_message_id)
\ No newline at end of file
+        super(HATestFailedError, self).__init__(rc=_rc, desc=_desc, message_id=_message_id)
+
+class SupportBundleError(HAError):
+    def __init__(self, desc=None):
+        """
+        Handle unimplemented function error.
+        """
+        _desc = f"Unimplemented Feature. stack: {inspect.stack()[1]}" if desc is None else desc
```

Comment:
Didn't get it. What may be unimplemented here ?

Reply:
It got copy/paste from UnimplementedError corrected it in next commit

Fixed